requirements.txt

```diff
@@ -37,3 +37,4 @@ wandb>=0.16.4
 xdg-base-dirs>=6.0.1
 psutil>=6.0.0
 huggingface_hub[hf_transfer]>=0.1.8
+haystack-ai>=2.8
```

Comment:
I am pretty sure new requirements like these need to be run by a specific set of people.

Reply:
could you please elaborate more on the "specific set of people" part? (for sure I miss something here)